src/ResourceManager/LogicApp/Commands.LogicApp.Test/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,50 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Xunit;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Microsoft.Azure.Commands.LogicApp.Test")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
```

Comment:
[Fixed] : Added Description and Company.

From: Mark Cowlishaw [mailto:notifications@github.com]
Sent: Monday, February 1, 2016 2:02 PM
To: Azure/azure-powershell <azure-powershell@noreply.github.com>
Cc: Pankaj Singh Negi <Pankaj.Negi@microsoft.com>
Subject: Re: [azure-powershell] Azure Logic App PowerShell Cmdlets (#1753)


In src/ResourceManager/LogicApp/Commands.LogicApp.Test/Properties/AssemblyInfo.cs<https://github.com/Azure/azure-powershell/pull/1753#discussion_r51488292>:

> +// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.

> +// See the License for the specific language governing permissions and

> +// limitations under the License.

> +// ----------------------------------------------------------------------------------

> +

> +using System.Reflection;

> +using System.Runtime.InteropServices;

> +using Xunit;

> +

> +// General Information about an assembly is controlled through the following

> +// set of attributes. Change these attribute values to modify the information

> +// associated with an assembly.

> +[assembly: AssemblyTitle("Microsoft.Azure.Commands.LogicApp.Test")]

> +[assembly: AssemblyDescription("")]

> +[assembly: AssemblyConfiguration("")]

> +[assembly: AssemblyCompany("")]

Please fill out as per other assemblies

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-powershell/pull/1753/files#r51488292>.